### PR TITLE
Update splash

### PR DESCRIPTION
**Pull request by aslakhol**
Add link to "Ditt liv i Online" wiki page.

Update dates.

Minor changes to wording

### CODE REVIEW

**Review by plusk — COMMENTED** [review]
Looks alright, but I'd like to see the API_EVENTS_URL updated. That will empty out the "previous events" section until the new velkom fills in new events.
(see: https://github.com/dotkom/glowing-fortnight/blob/d6ffd4b10173ad70ffc39aabd47b4c52554ccc24/common/constants.js#L1)

**Review by hernil — APPROVED** [review]

File: components/join/index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+      <h1>Bli aktiv i linjeforeningen!</h1>
       <p>Online er linjeforeningen for alle informatikere. Gå videre til <a href="https://online.ntnu.no/">hovedsiden</a> for å lage brukerkonto og bli medlem.</p>
+      <p>Lurer du på hva som skjer i linjeforeningen eller hvem du skal kontakte for å få booke kontoret? En kjapp innføring i ditt liv i Online er å finne på <a href="https://online.ntnu.no/wiki/online/info/sosialt-og-okonomisk/ditt-liv-som-onliner/" target="_blank">wikien</a>.</p>
       <p>Har du lyst til å gjøre studietiden til informatikere enda bedre? Som komitemedlem i Online blir du del av en familie du aldri vil glemme. Vi tilbyr utfordrende og spennende verv i et meget sosialt miljø med stor takhøyde.</p>
       <p>Les mer om de ulike komiteene på <a href="https://opptak.online.ntnu.no/" target="_blank">opptakssiden vår</a>.</p>
-      <p>Søknadsfrist for komiteopptak er 27. august kl. 23:59. Søknader rettes til <a href="mailto:opptak@online.ntnu.no">opptak@online.ntnu.no</a></p>
+      <p>Søknadsfrist for komiteopptak er 26. august kl. 23:59. Søknader rettes til <a href="https://opptak.online.ntnu.no/" target="_blank">opptakssiden</a>.</p>
```
Comment (duvholt) [inline components/join/index.js:9]: Maybe use our sweet domain opptak.online instead?

**Review by duvholt — APPROVED** [review]

**Review by plusk — APPROVED** [review]
Awesome diff. Great size. Look thick. Solid. Tight. Keep us all posted on your continued progress with any new commits or pull requests. Show us what you got man. Wanna see how freakn’ huge, solid, thick and tight your contributions stats can get. Thanks for the motivation

**Review by sklirg — APPROVED** [review]

**Comment by oleast** [comment]
Updated the logos for this year, should I also remove the images which are not used anymore?

splash_logos

**Comment by duvholt** [comment]
IMO the images can stay. It's likely that they will be reused later.

**Comment by oleast** [comment]
Would also advise against keeping the lock file this PR adds, as it contains really outdated versions of some packages, most notably `node-sass`.
This led to problems with building the project locally, as the package requires a binary which is no longer available (node-sass bindings for version 3.8). This can be built automatically with node-gyp as part of the normal dependency build process, but it failed on the systems I tried on.

There are also a lot of security concerns related to other packages in this project, but from what I can see they are mostly not relevant to their usage here?

This project is worked on quite rarely, and I think it would be good to address these things sooner rather than later. If not, it may only get worse next year.¨

Edit: Downgrading from version 10.5 to 9.11 of `node` allowed me to build correctly.

**Comment by duvholt** [comment]
We also already have a yarn lockfile.  Keeping a lockfile in itself is not a problem as it can easily be updated with the latest valid versions. Though keeping two different lockfiles is really useless and invalidates the reason to use a lockfile.

**Pull request merged**